common/ux-window.cpp

```diff
@@ -623,13 +633,14 @@ namespace rs2
 
         // reset graphic pipe
         begin_frame();
-
+        
```

Comment:
Please try to avoid unneeded changes
Here and on line 643

Reply:
fixed